topics/gh-extension/index.md

```diff
@@ -0,0 +1,12 @@
+---
+created_by: GitHub
+display_name: gh extension
+github_url: https://github.com/cli/cli
+logo: gh-extension.png
+released: September 16th, 2020
+short_description: Extensions to the GitHub CLI
+topic: gh-extension
+url: https://cli.github.com
+---
+gh is GitHub's official CLI client and it can be extended by anyone through its extension system. Repositories with this topic are able to be installed, giving gh new behaviors.
+=
```

Comment:
Is the `=` special syntax for something?

Reply:
I'm not sure what it does either, even though I've seen it before in markdown. Maybe a horizontal rule? Let's remove it.
```suggestion
```